js/guideposts.js

```diff
@@ -409,6 +412,23 @@ osmcz.guideposts = function (map, baseLayers, overlays, controls, group) {
             // PhotoDB button (add an image)
             if (!photoDBbtn) {
                 photoDBbtn = L.control.photoDbGui().addTo(map);
+
+                osmcz.fodyFilterEl = document.createElement("div");
+                osmcz.fodyFilterEl.innerHTML = "<div style='position:absolute;top:83px;'>" +
+                    "<input type='checkbox' value='gp'> rozc<br><input type='checkbox' value='table'> tab<br><input type='checkbox' value='other'> jine</div>"
+                document.getElementsByClassName("leaflet-control-photoDBbtn")[0].appendChild(osmcz.fodyFilterEl)
+                osmcz.fodyFilterEl.addEventListener("click", function(e){
+                    e.stopPropagation(); // do not trigger the photodb "add image" button
+                    const checkboxes = osmcz.fodyFilterEl.getElementsByTagName("input");
+                    const filter = [];
```

Comment:
one-liner:
```
osmcz.fodyFilter = checkboxes.filter(ch => ch.checked).map(ch => ch.value);
```

;-)

Reply:
No, `checkboxes` je HTMLCollection, takže by bylo potřeba to první zkonvertovat na pole. A byl jsem líný hledat jak se to dělá, vím že to jde :)


<img src="https://user-images.githubusercontent.com/385047/131826551-bf997016-40a3-4332-9e66-16ad72cf2377.png" width=300>

Reply:
A co třeba `$('#xxx input:checked')` ?

Reply:
@mkyral jo, velice elegantní! 

Mariáne, jmenuji tě novým vlastníkem tohoto PR. Všechna sláva za dokončení nechť padne na tvou hlavu 😅👍🖖

Reply:
@mkyral jo, velice elegantní! 

Mariáne, jmenuji tě novým vlastníkem tohoto PR. Všechna sláva za dokončení nechť padne na tvou hlavu 😅👍🖖

Reply:
> No, `checkboxes` je HTMLCollection, takže by bylo potřeba to první zkonvertovat na pole.

aha, jasne. Potom `osmcz.fodyFilter = Array.from(checkboxes).filter(ch => ch.checked).map(ch => ch.value);`. Ale vidim ze uz mate jquery riesenie :-).